hack/verify-deps.sh

```diff
@@ -0,0 +1,12 @@
+#!/bin/bash
+set -euo pipefail
+
+function print_failure {
+  echo "There appear to be uncommitted changes to go.mod. Please run 'go tidy' and 'go vendor'."
```

Comment:
Instructions should be `go _mod_ {tidy,vendor}`

Reply:
And, arguably, listed as `vendor`, then `tidy`.

Reply:
Fixed